b3sum/src/lib.rs

```diff
@@ -0,0 +1,599 @@
+use anyhow::{bail, ensure, Error, Result};
+use std::cmp;
+use std::fs::File;
+use std::io;
+use std::io::prelude::*;
+use std::iter::from_fn;
+use std::path::{Path, PathBuf};
+use std::str::FromStr;
+
+#[cfg(test)]
+mod unit_tests;
+
+pub const NAME: &str = "b3sum";
```

Comment:
I think `NAME` belongs to `main.rs`, it doesn't make much sense to be part of `lib`